HerPublicWebsite/appsettings.json

```diff
@@ -27,6 +27,14 @@
             "LocalAuthorityNamePlaceholder": "local authority name",
             "LocalAuthorityWebsiteUrlPlaceholder": "local authority website url"
         },
+        
+        "ReferenceCodePendingLATemplate": {
+            "Id": "287f1654-c01c-4269-a03b-34b87db56b4a",
+            "RecipientNamePlaceholder": "recipient name",
+            "ReferenceCodePlaceholder": "reference code",
+            "LocalAuthorityNamePlaceholder": "local authority name",
+            "LocalAuthorityWebsiteUrlPlaceholder": "local authority website url"
+        },

```

Comment:
In the actual name of the template on GOV.UK Notify, can you remove the apostrophe in "LA's" please?

The correct pluralisation is just "Reference code (Pending LAs)".

![image](https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/assets/3533273/9918877b-7543-4025-9091-ee84fd83b81f)